circuits/chunkedmul.circom

```diff
@@ -134,10 +135,10 @@ template BinMulFastChunked51(m, n){ //base 2**51 multiplier
   
   carry[0] <== 0;
   for (j=0; j<m+n-1; j++){
-    out[j] <-- (sum[j]+carry[j])%2251799813685248;
-    carry[j+1] <-- (sum[j]+carry[j])\2251799813685248;
+    out[j] <-- (sum[j]+carry[j])%power;
+    carry[j+1] <-- (sum[j]+carry[j])\power;
     //Note: removing this line does not change the no of constraints
-    sum[j]+carry[j] === carry[j+1]*2251799813685248 + out[j];
+    sum[j]+carry[j] === carry[j+1]*power + out[j];
```

Comment:
While at it could you also fix the spacing around the operator? 

`out[j] <-- (sum[j] + carry[j]) % power;`